### ISSUE TITLE
Add a Save button below the License Ownership information fields

### ISSUE THREAD

**Issue by AndreeaChi** [xwikisas/application-licensing#130]
Hi! It would be more user friendly to have a Save button under the License Ownership fields in the Licenses administration.

Currently, the only option to save those details is to click on the Buy, Get trial or Extend license buttons, next to an installed Pro application shown in the Licenses table.

![image](https://user-images.githubusercontent.com/36840106/195565095-370c3988-9ad3-4e3a-8ace-09cd558a74ec.png)

**Comment by mflorea** [comment]
Yes, but why would you fill those fields if not to be able to buy a license or get a trial one? What workflow do you have in mind? Filling license ownership information and then leaving the page doesn't make much sense no? The Save button you propose might make things clearer but it also adds an additional click.

**Comment by mflorea** [comment]
OK, so the use case is changing / updating the license ownership information. I agree it's a valid use case.
